.travis/osx..script.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
         # Workaround; No FindQt5.cmake module exists
-        export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        export CMAKE_PREFIX_PATH
 fi
 
-cmake $CMAKE_FLAGS -DUSE_WERROR=OFF ..
+cmake "$CMAKE_FLAGS" -DUSE_WERROR=OFF ..
```

Comment:
I mean, I hope Travis will fail so I know it's wrong. If it pass, then.. further investigation needed.